### PR TITLE
Initial update to SIG Release teams for 1.16

### PR DESCRIPTION
**Pull request by justaugustus**
Signed-off-by: Stephen Augustus <saugustus@vmware.com>

/assign @calebamiles @tpepper @lachie83 
/cc @kubernetes/owners @kubernetes/release-team-leads @kubernetes/release-team

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -169,34 +170,38 @@ teams:
...
     - nickchase # 1.15 Release Notes Shadow
-    - nikopen # 1.15 RT Lead Shadow
+    - nikopen # 1.16 RT Lead Shadow
+    - onlydole # 1.16 Communications Lead
     - onyiny-ang # 1.15 Release Notes Lead
+    - saschagrunert # 1.16 Release Notes Lead
+    - simplytunde # 1.16 Docs Lead
     - slicknik # 1.15 Branch Manager Shadow
     - smourapina # 1.15 CI Signal Shadow
     - soggiest # 1.15 Bug Triage Lead
```
Comment (soggiest) [inline config/kubernetes/sig-release/teams.yaml]: You should remove me from Bug Triage lead

Comment (justaugustus) [reply config/kubernetes/sig-release/teams.yaml]: @soggiest -- planning a follow-up PR once we lock the shadows in. You'll probably be shuffling teams, so it doesn't make sense to remove you just yet :)

**Pull request merged**